imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc

```diff
@@ -34,7 +34,16 @@ See the [KYC API endpoint references](https://docs.immersve.com/api-reference/ky
 
 ## Submit cardholder KYC statement
 [Submit KYC statement](https://docs.immersve.com/api-reference/submit-partner-kyc-statement#submit-partner-kyc-statement) about the cardholder account.
-Provide "passall" as the middle name, the check should be successful regardless of the validity of the rest of the details.
+
+{% note %}
+When testing, to improve chances of a successful response, please provide "passall" as the middleName, and generate a random givenName, familyName, and dateOfBirth.
```

Comment:
is it possible for the check to fail with `passall`? I'm wondering if we should use a more definitive language, like `to ensure a successful response` or something similar instead of `to improve chances`

Reply:
it's a requirement from Frankie one for passall to pass 1 aspect of the validation. However if there is coincidentally someone at the same or popular address and you use a common name etc. then you can still receive a failure.

Reply:
yeah, we can't guarantee a success, so 'ensure' would be an overstatement

Reply:
I see. Thanks for the clarification!